### PR TITLE
fix: thorchain LP handle 0-value position page

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Spotted in https://github.com/shapeshift/web/pull/6373#pullrequestreview-1923174916:

> Withdraw succesful ✅ though noted that after a 100% withdraw, the pool page is gone and we're left with a big dark screen of void 😢 because we refetch user data, which is now none. Will tackle in a follow-up to handle 0'd out values on pool position page

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

> What protocols, transaction types or contract interactions might be affected by this PR?

None, only handles no position as valid and adds skeletons while at it

## Testing

- Exit fully from a position
- After the Tx completes, ensure the position goes to a 0-value position instead of becoming empty

### Engineering

- ^

### Operations

- ^

## Screenshots (if applicable)

<div class='graphite__hidden'>
          <div>🎥 Video uploaded on Graphite:</div>
            <a href="https://app.graphite.dev/media/video/BpXudHfKodngd8bjaa8L/6414e4fe-4a6a-47a5-bc63-f5949aae9de1.mov">
              
            </a>
          </div>
<video src="https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/BpXudHfKodngd8bjaa8L/6414e4fe-4a6a-47a5-bc63-f5949aae9de1.mov">Screen Recording 2024-03-07 at 12.06.26.mov</video>

### CODE REVIEW

**Comment by gomesalexandre** [comment]
* **#6378** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6378?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gomesalexandre and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/pages/ThorChainLP/Position/Position.tsx [hunk 1]
```diff
@@ -191,7 +194,8 @@ export const Position = () => {
     [stepIndex],
   )
 
-  if (!position) return null
+  console.log({ pool, position })
```
Comment (gomesalexandre) [inline src/pages/ThorChainLP/Position/Position.tsx:197]:
```suggestion
```

File: src/pages/ThorChainLP/Position/Position.tsx [hunk 2]
```diff
@@ -191,7 +194,7 @@ export const Position = () => {
     [stepIndex],
   )
 
-  if (!position) return null
+  if (!pool) return null
```
Comment (kaladinlight) [inline src/pages/ThorChainLP/Position/Position.tsx:197]:
```suggestion
```

Comment (gomesalexandre) [reply src/pages/ThorChainLP/Position/Position.tsx]: 58d4c669c786938b801ae540dbcd143820cc8720

File: src/pages/ThorChainLP/Position/Position.tsx [hunk 3]
```diff
@@ -201,7 +204,7 @@ export const Position = () => {
             <CardHeader px={8} py={8}>
               <Flex gap={4} alignItems='center'>
                 <PoolIcon assetIds={poolAssetIds} size='md' />
-                <Heading as='h3'>{position.name}</Heading>
+                <Heading as='h3'>{pool.name}</Heading>
```
[the remark this answers is not in the document]
Comment (gomesalexandre) [reply src/pages/ThorChainLP/Position/Position.tsx]: 58d4c669c786938b801ae540dbcd143820cc8720 with added skeleton

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**